src/Illuminate/Encryption/Encrypter.php

```diff
@@ -374,4 +375,30 @@ public function previousKeys(array $keys)
 
         return $this;
     }
+
+    /**
+     * Re-encrypt the given payload using the current primary key.
+     *
+     * @param  string  $payload
+     * @param  bool  $unserialize
+     * @return string
+     */
+    public function reEncrypt(string $payload, bool $unserialize = true): string
+    {
+        if (empty($payload)) {
```

Comment:
Since this is typed as a `string`, you can avoid the overhead of calling a method here, since an empty state can only appear in one case here: 
```suggestion
        if ($payload === '') {
```

Reply:
Yeah, that makes sense. Thanks for pointing it out.